mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -322,6 +322,30 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 					},
 				},
 			},
+			"advanced_options_config": {
+				Type: schema.TypeList,
+				Optional: true,
+				Description: `Advanced Options Config of this security policy.`,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"json_parsing": {
+								Type: schema.TypeString,
+								Optional: true,
+								Default: "STANDARD",
```

Comment:
I'm now a little confused as well :) but if `DISABLED` is the default, we probably want that to match here.

Reply:
You're right - got it. Updated!

Also checking with someone on the GCP Slack to verify that I'm understanding correctly which is which, but if you've got any contacts internally at Hashi or Google who can help, that would also be great.